reactive/kotlinx-coroutines-reactor/src/FlowAsFlux.kt

```diff
@@ -0,0 +1,21 @@
+package kotlinx.coroutines.reactor
```

Comment:
We have `Flow.asPublisher` in `kotlinx.coroutines.reactive.flow` and `asFlux` in `kotlinx.coroutines.reactor`.
We should add or remove `flow` prefix and I'm thinking about its removal.
WDYT @elizarov ?

Reply:
Removed the `flow` prefix.